dags.yaml

```diff
@@ -1343,3 +1343,19 @@ bqetl_glam_export:
   tags:
     - impact/tier_2
   description: DAG to prepare GLAM data for public export.
+
+bqetl_crash:
+  schedule_interval: 0 2 * * *
+  default_args:
+    depends_on_past: false
+    email:
+      - dthorn@mozilla.com
+      - telemetry-alerts@mozilla.com
+    email_on_failure: false
+    email_on_retry: true
+    owner: dthorn@mozilla.com
```

Comment:
I will take ownership of this with me to observability team, but until time is available for some feature work bigquery-etl and airflow still seem the best place for it to run.